src/Text/LLVM/AST.hs

```diff
@@ -342,6 +355,14 @@ parseDataLayout str =
            'm' -> Mangling      <$> (char ':' >> pMangling)
            _   -> mzero
 
+    pFunctionPointerAlignType :: Parser FunctionPointerAlignType
+    pFunctionPointerAlignType =
+      do c <- letter
+         case c of
+           'i' -> return IndependentOfFunctionAlign
+           'n' -> return IndependentOfFunctionAlign
```

Comment:
Should this be `'n' -> return MultipleOfFunctionAlign` ?

Reply:
Oops, good catch! That is a copy-paste mistake, and it should indeed be `MultipleOfFunctionAlign`.